.github/workflows/lint.yml

```diff
@@ -32,17 +32,21 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5.1.0
       with:
-        go-version-file: 'go.mod'
+        go-version: '1.22'
         check-latest: true
       id: go
       if: ${{ !contains(matrix.os, 'ARMHF') }}
     - id: govulncheck
       uses: golang/govulncheck-action@b625fbe08f3bccbe446d94fbf87fcc875a4f50ee # v1.0.4
       with:
         cache: false
-        go-version-input:
-        go-version-file: 'go.mod'
+        # go-version-file do not account for latest compatible go https://github.com/actions/setup-go/issues/481
+        # which will break https://github.com/kopia/kopia/issues/4292 requirement if used
+        # leave below commented for vuln check to ignore vulns fixed in latest go
+        # go-version-file: 'go.mod' 
+        go-version-input: '1.22'
```

Comment:
This fixes the vulncheck that [previously failed](https://github.com/kopia/kopia/actions/runs/12328329173/job/34421677211?pr=4303)

https://github.com/kaovilai/kopia/actions/runs/12337854810/job/34432101853?pr=2

See following issues for usability enhancements:
- https://github.com/actions/setup-go/issues/481
- https://github.com/golang/go/issues/70853